### PR TITLE
[Feat] Add startup call for isochrone calculation

### PR DESCRIPTION
**Pull request by metemaddar**
Used for compile numba for the first time.
Related issue: #1521

### CODE REVIEW

**Comment by metemaddar** [comment]
I'm going to add also the time middleware to check the timing for isochrone API.

**Comment by metemaddar** [comment]
@majkshkurti  For walking it works. But for cycling not:

```
INFO:     Waiting for application startup.
App is starting...
First run of isochrones....
walking
Execution time: 26.659935235977173 seconds
cycling
Execution time: 21.875360250473022 seconds
Running isochrones first call complete!
INFO:     Application startup complete.
```
#### Call walking again
Execution time: **7.415709018707275** seconds
#### Call cycling again
Execution time: **23.66732931137085** seconds
Execution time: **22.01812744140625** seconds

**Comment by majkshkurti** [comment]
Walking is enough for triggering the compilation. So single and multi, as mode doesn't matter

File: app/api/src/run_time_method_calls.py [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+        "output": {"type": "grid", "resolution": "9"},
+    },
+}
+
+
+async def run_calculate_isochrone_single_default(
+    client: httpx.AsyncClient, superuser_token_headers: Dict[str, str]
+) -> None:
+    # data["routing_profile"] = "single_walking_default"
+    for mode in ["transit", "walking"]:
```
Comment (majkshkurti) [inline app/api/src/run_time_method_calls.py:58]: Can we also add multi isochrone calculation here ?

**Review by majkshkurti — COMMENTED** [review]
Thanks @metemaddar

File: app/api/src/run_time_method_calls.py (hunk 1 above)
Comment (metemaddar) [reply app/api/src/run_time_method_calls.py]: Yes. Done.

**Review by majkshkurti — APPROVED** [review]

**Pull request merged**